run_RF2NA.sh

```diff
@@ -16,8 +16,8 @@ SCRIPT=`realpath -s $0`
 export PIPEDIR=`dirname $SCRIPT`
 HHDB="$PIPEDIR/pdb100_2021Mar03/pdb100_2021Mar03"
 
-CPU="8"  # number of CPUs to use
-MEM="64" # max memory (in GB)
+CPU="128"  # number of CPUs to use
+MEM="500" # max memory (in GB)
```

Comment:
can you revert this change?

Reply:
Hi @fdimaio,
I have set the CPU and MEM resources to the default settings.